src/components/chart/_macro.njk

```diff
@@ -104,7 +104,7 @@
         {%
             set config = {
                 "legend": {
-                    "enabled" : params.legend if params.legend else true
+                    "enabled" : false if params.legend == false else true
```

Comment:
This logic would be true if anything is passed into params.legend as a string even "false" or "fghdakgfhdsgfhd". I think this logic should be a bit more explicit to stop this. Also in a few of the examples what is passed into this is a string and some a bool. They should all be a bool.

Reply:
Update: it looks like I can check if the parameter is defined or not with `{% if params.legend is defined %}`, and testing suggests this works in Wagtail too. cc @nimasmi .

Reply:
Good spot. I wasn't aware of this, but it is part of Jinja2 (see [Test syntax docs](https://jinja.palletsprojects.com/en/stable/templates/#tests) and [`jinja_tests.defined()`](https://jinja.palletsprojects.com/en/stable/templates/#jinja-tests.defined)).

Reply:
This should now be more robust in its checking, and should still work OK in Wagtail. @nimasmi @janehughes

Reply:
I've realised the `===` has crept back in which won't work in Wagtail, so I need to address this.

Reply:
Sorted.